packages/account/src/Components/Routes/helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { matchPath, RouteProps } from 'react-router';
+import { routes } from '@deriv/shared';
+import { TRouteConfig } from 'Types';
+
+export const normalizePath: (path: string) => string = path => (/^\//.test(path) ? path : `/${path || ''}`); // Default to '/'
+
+export const findRouteByPath = (path: string, routes_config?: TRouteConfig[]): RouteProps | undefined => {
```

Comment:
it is better to set a type and some default value for returning variable, for example empty string , then use indefined as a type